### DISCUSSION TITLE
What are these numbered 'tabs' up the top right?

### DISCUSSION THREAD

**Discussion by ToddMurphy92** [LazyVim/LazyVim#4970] [Q&A] [answered]
Can anyone tell me what this feature is called that opens up these two numbered 'tabs' in the top right?
![image](https://github.com/user-attachments/assets/cf35be9b-82fd-44dd-9477-89541886a3e8)

Clicking them switches me between to different 'sessions' of neovim. I've no idea how I created the second one, or what these are called but it seems like a neat feature that I would like to make use of.

**Comment by dpetka2001** [comment, marked as answer]
They are normal Neovim tabs. The others you see are just buffers and it's the `bufferline.nvim` plugin that makes the buffers look like tabs in other editors and browsers (but it's not native Neovim behavior). More info can be found at `:h window`.

You probably created one by pressing `<leader><tab><tab>` or `:tabnew` command.

**Comment by ToddMurphy92** [comment]
Thanks @dpetka2001 - much appreciated the response and I'm enjoying learning the related keymaps now.
